stdlib/sched.pyi

```diff
@@ -1,23 +1,26 @@
 import sys
 from collections.abc import Callable
 from typing import Any, NamedTuple
+from typing_extensions import TypeAlias
 
 __all__ = ["scheduler"]
 
+_ActionCallback: TypeAlias = Callable[..., Any]
```

Comment:
```suggestion
_ActionCallback: TypeAlias = Callable[..., object]
```
Looks like the return value is ignored

Reply:
It's used as an attribute annotation as well, though, and I wanted to avoid unexpected false positives on code like

```python
e: Event
x = e.action()
x.foo()
```

I'm not sure if it makes sense to do that, but I didn't want to cause a regression. Same for `_WriterCallback` in `wsgiref.validate`.